src/Util.php

```diff
@@ -201,30 +201,4 @@ public static function convertDateTimeObject(\DateTime $dateTime, bool $includeT
 
         return $dateTime->format($formatString);
     }
-
-    /**
-     * Converts Request to Curl console command.
-     *
-     * @return string
-     */
-    public static function convertRequestToCurlCommand(Request $request)
```

Comment:
This was for debugging purpose if I remember correctly. Not compatible anymore?

Reply:
yes, it was used in utils test function testConvertRequestToCurlCommand but this function was not used anywhere in the tests.